signbank/dictionary/tasks.py

```diff
@@ -77,16 +68,27 @@ def retrieve_videos_for_glosses(video_details: List[VideoDetail]):
     videos_to_create = []
 
     temp_dir = TemporaryDirectory(dir=settings.MEDIA_ROOT)
-    if settings.GLOSS_VIDEO_FILE_STORAGE == "storages.backends.s3boto3.S3Boto3Storage":
-        temp_dir = TemporaryDirectory()
+
+    s3 = boto3.client("s3")
 
     for video in video_details:
         retrieval_url = f"{settings.NZSL_SHARE_HOSTNAME}{video['url']}"
-        file_name = f"{temp_dir.name}/{video['file_name']}"
-        file, _ = urlretrieve(
-            retrieval_url,
-            file_name
-        )
+
+        if settings.GLOSS_VIDEO_FILE_STORAGE == "storages.backends.s3boto3.S3Boto3Storage":
```

Comment:
**suggestion**: given this conditional is evaluated with string literals multiple times, might it be worth comparing once, earlier and assigning it to a boolean?